src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './FindMovie.scss';
+import classnames from 'classnames';
+import { getMovie } from '../../api';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
+
+type Props = {
+  setMovies: (value: any) => void,
```

Comment:
try to fix 
![image](https://user-images.githubusercontent.com/70009205/207127999-a291d33b-6aa8-4a51-8113-5ab274baef62.png)

Reply:
couldn't handle the problem so i changed the logic a bit